### PR TITLE
Use "full" flavor on parallel builds

### LINKED ISSUE #13431: Add a "full" build
Currently we have two pairs of build. Amazon and Play.
Amazon does not allow to take picture. Play, soon, won't allow to throw to access all media. So we need a third build that has no artificial limitation. I'd suggest a name such as "full", given it's not related to a specific store, but just to have the full features.

This probably require to change ./tools/release.sh, so that this build is published on github.

Note: handle the backup prompt for full build as mentioned here: https://github.com/ankidroid/Anki-Android/pull/13424/commits/88cde70490430a7ba3c5843b15deb1108d2db029#r1147041922

### PR DESCRIPTION
**Pull request by BrayanDSO**
## Fixes
Fixes #13431

## Approach
Change the assemble type on the script

## How Has This Been Tested?

Runned `./gradlew assembleFullRelease` to check if the command was okay. Didn't run the script because of the credentials.

## Checklist
_Please, go through these checks before submitting the PR._

- [X] You have a descriptive commit message with a short title (first line, max 50 chars).
- [X] You have commented your code, particularly in hard-to-understand areas
- [X] You have performed a self-review of your own code
- [ ] UI changes: include screenshots of all affected screens (in particular showing any new or changed strings)
- [ ] UI Changes: You have tested your change using the [Google Accessibility Scanner](https://play.google.com/store/apps/details?id=com.google.android.apps.accessibility.auditor)

### CODE REVIEW

**Comment by david-allison** [comment]
This means the regular APK and the arch specific APKs are using `play`, but the parallel alphas are using `full`.

We should have a quick discussion on whether this is the right move as we'll need to (slightly) change documentation

**Comment by mikehardy** [comment]
I believe all APKs should use full *except* the ones we upload to play store. I do not want to throw a bunch of flavors up on github releases (they're pretty chunky, feels like abuse of storage service to me...) so that implies we pick one, and "full" is the one you'd want to pick. No reason to restrict a manually installed APK in any way, IMHO

**Comment by BrayanDSO** [comment]
So, `release.sh` should be changed to make a "full" and a "play" non-parallel build and upload the "full" one to GitHub. Probably for another PR

**Comment by mikehardy** [comment]
I was planning on digging in to this in a little bit but you're already in here and this change looks good :-), I think it should just go in here in another commit

Change `play` here to `full` https://github.com/ankidroid/Anki-Android/blob/7ab289a0ee3ed114570cf1554b89c2cb3ca2e903/tools/release.sh#L142

Change this:
https://github.com/ankidroid/Anki-Android/blob/7ab289a0ee3ed114570cf1554b89c2cb3ca2e903/tools/release.sh#L130-L133

to "now generate the full flavor builds for all the architectures and universal" as a comment, alter `play` to full in the gradle target and.....that should be it?

**Comment by mikehardy** [comment]
(if it was more complicated I'd say yeah another PR but this should be that easy 🤞 )

File: tools/parallel-package-release.sh [hunk 1]
```diff
@@ -33,7 +33,7 @@ BUILDNAMES='A B C D E'
 for BUILD in $BUILDNAMES; do
     LCBUILD=`tr '[:upper:]' '[:lower:]' <<< $BUILD`
     ./gradlew --stop
-    if ! ./gradlew assemblePlayRelease -PcustomSuffix="$LCBUILD" -PcustomName="AnkiDroid.$BUILD" -Duniversal-apk=true
+    if ! ./gradlew assembleFullRelease -PcustomSuffix="$LCBUILD" -PcustomName="AnkiDroid.$BUILD" -Duniversal-apk=true
```
Comment (david-allison) [inline tools/parallel-package-release.sh:36]: Line 41 needs changing to `cp AnkiDroid/build/outputs/apk/full/release/AnkiDroid-full-universal-release.apk ./AnkiDroid-$TAG.parallel.$BUILD.apk
done`

**Review by david-allison — CHANGES_REQUESTED** [review]
One issue, rest LGTM

**Review by david-allison — APPROVED** [review]

**Review by mikehardy — APPROVED** [review]
Looks right to me, will squash. Thanks @BrayanDSO ! Knocked a thing off my todo list so helps me personally not just project, not that anyone else should care but I do haha :-)

**Comment by Arthur-Milchior** [comment]
Thanks @BrayanDSO.

> I believe all APKs should use full _except_ ...

I'm not entirely certain. In the sense that, if we want to ask users to alpha tests the migration, they should be able to download it from github (at least, if I understand correctly that we want to test with github alpha testers first, before testing with play store ones)

**Comment by mikehardy** [comment]
May be that we want a specific "play" build on the releases page as well for testing?
I believe "full" should be the default for all non-play store locations (read as: github releases page) but you have a point @Arthur-Milchior about testing play release from the releases page. Could do a single universal play variant there. In fact, could do a universal version of each non-full variant on the releases page, for testing (read as: play + amazon universal builds)

If you like that idea, peel this to an issue and assign me, I can get it

**Pull request merged**